From 647d2e1849f7360bd0cad532e21cf63ec294f4ec Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Tue, 16 Oct 2018 13:28:45 -0500
Subject: [PATCH 4879/5242] MLK-19955: arm64: dts: fix the qm flexspi clock
 name and power domain

commit  71855f52d2678a33e87d4722145066ff7410b41a from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct the clock name and power domain for i.MX8QM Flexspi.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 0972f7b..d86fc0d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -4059,10 +4059,10 @@
 			<0x0 0x08000000 0x0 0x19ffffff>;
 		reg-names = "FlexSPI", "FlexSPI-memory";
 		interrupts = <GIC_SPI 92 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&clk IMX8QM_FSPI0_CLK>,
-		<&clk IMX8QM_FSPI0_CLK>;
-		assigned-clock-rates = <29000000>,<29000000>;
-		clock-names = "qspi_en", "qspi";
+		clocks = <&clk IMX8QM_FSPI0_CLK>;
+		assigned-clock-rates = <29000000>;
+		power-domains = <&pd_lsio_flexspi0>;
+		clock-names = "fspi";
 		status = "disabled";
 	};
 
-- 
1.7.9.5

